high_card.rb

```diff
@@ -1,5 +1,29 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
+n = 0
 players = []
+puts "{n} players so far. Enter a player name, or type 'play':"
+def get_players
+  player_array = []
+  while gets.chomp != 'play'
+      new_player = gets.chomp
+      new_player >> player_array
```

Comment:
In ruby `<<` is a method called 'the shovel' that works very similarly to .push and sometimes like addition. It doesn't work if you have it the other way around.